File: MultiLineSearch/Guids.vb
Imports System

Class GuidList
    Private Sub New()
    End Sub

    Public Const GUID_MULTI_LINE_SEARCH_PKG_STRING As String = "29956436-0205-4c9e-9396-abce099383f9"
    Public Const GUID_MULTI_LINE_SEARCH_CMD_SET_STRING As String = "7d58fc84-2160-4645-be98-eb7f27c358f7"
    Public Const GUID_TOOL_WINDOW_PERSISTANCE_STRING As String = "5c82a791-7c9f-441e-952b-8848655dd85e"

    Public Shared ReadOnly GuidMultiLineSearchCmdSet As New Guid(GUID_MULTI_LINE_SEARCH_CMD_SET_STRING)
End Class

File: MultiLineSearch2/Guids.cs
// Guids.cs
// MUST match guids.h
using System;

namespace Helixoft.MultiLineSearch
{
    static class GuidList
    {
        public const string GUID_MULTI_LINE_SEARCH_PKG_STRING = "29956436-0205-4c9e-9396-abce099383f9";
        public const string GUID_MULTI_LINE_SEARCH_CMD_SET_STRING = "7d58fc84-2160-4645-be98-eb7f27c358f7";
        public const string GUID_TOOL_WINDOW_PERSISTANCE_STRING = "5c82a791-7c9f-441e-952b-8848655dd85e";

        public static readonly Guid GuidMultiLineSearchCmdSet = new Guid(GUID_MULTI_LINE_SEARCH_CMD_SET_STRING);
    };
}